src/app/test-results/test-results-list/test-results-detail-list-item.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component, Input } from '@angular/core';
+import { ResponseTestResultDetail } from 'api-client';
+
+@Component({
+  selector: 'wh-test-results-detail-list-item',
+  templateUrl: './test-results-detail-list-item.component.html',
+})
+export class TestResultsDetailsListItemComponent {
```

Comment:
When adding components to the project with the angular cli tool ex. `npx ng generate component test-results-list/TestResultsDetailsListItem`  (or short  `ng g c test-results-list/TestResultsDetailsListItem`) the files (html, scss, spec, ts) will be generated for you. In this case i see good cause for removing an otherwise empty scss file, the .spec file however could in its default form be usable without adding further tests it would still ensure you component works for runtime. Additionally having the component own a test file will more accurately describe test coverage in system where test coverage is designated by the number of lines touched where tests have been started.

Reply:
Hmm, I see...
I haven't been using any commands to make these files, so that's really good to know 😅